### PR TITLE
Makeup

### PR DESCRIPTION
**Pull request by ceremcem**
Attempt to make ractive.js.org more readable: 

## Current

image

## New 

image

### CODE REVIEW

**Comment by fskreuz** [comment]
Yeah, the grey bg was supposed to prevent excessive contrast against dark text but I guess it has its drawbacks.

Feel free to play with the colors as long as the green stays, and the docs remain plain and clean. I do have an aversion to shadows tho (and all things 3D-ish). 😁  Consider a darker grey bg and maroon for color similar to the code blocks for consistency.

**Comment by ceremcem** [comment]
Hmm. I'm opening a new branch if I'm not sure if the PR will be accepted or not, thus preventing a cherry-pick operation.

**Comment by ceremcem** [comment]
image

File: theme/theme.css [hunk 1]
```diff
@@ -35,12 +35,12 @@ blockquote{
...
 code{
   word-break: break-all;
-  background-color: #fafafa;
+  background-color: rgb(240,240,240);
   border-radius: 4px;
   padding: 3px 5px;
   margin: 4px;
   font-family: "courier";
-  box-shadow: 0.5px 0.5px 2px #888888;
+  color: maroon;
```
Comment (fskreuz) [inline theme/theme.css]: Never knew that color was really maroon. 😁

**Pull request merged**